geometry_msgs/Point[] points
---
std_msgs/Bool success